packages/models/test/fixtures/eslint-plugin.mock.ts

```diff
@@ -1,14 +1,44 @@
-import type { CategoryConfig, PluginReport } from '../../src';
+import type {
+  Audit,
+  CategoryConfig,
+  PluginConfig,
+  PluginReport,
+} from '../../src';
 import { ESLINT_AUDITS_MAP } from './eslint-audits.mock';
+import { runnerConfig } from './runner.mock';
+
+const eslintMeta = {
+  slug: 'eslint',
+  title: 'ESLint',
+  icon: 'eslint',
+  description: 'Official Code PushUp ESLint plugin',
+  packageName: '@code-pushup/eslint-plugin',
+  version: '0.1.0',
+} satisfies Partial<PluginConfig>;
+
+export function eslintPluginConfig(outputDir = 'tmp'): PluginConfig {
+  const audits = Object.values(ESLINT_AUDITS_MAP).map(
+    ({ slug, description, title, docsUrl }) =>
+      ({
+        slug,
+        description,
+        title,
+        docsUrl,
+      } satisfies Audit),
+  );
+  return {
+    ...eslintMeta,
+    runner: runnerConfig(
+      Object.values(ESLINT_AUDITS_MAP),
+      `${outputDir}/eslint-out.json`,
```

Comment:
For Windows compatibility:

```suggestion
      join(outputDir, 'eslint-out.json'),
```

(+ `import { join } from 'path'`)